app/models/user/tasks/message_expiry.rb

```diff
@@ -0,0 +1,16 @@
+module User
+  module Tasks
+    class MessageExpiry
+      def self.expire(uid)
```

Comment:
@developish - Why are we expiring so many things in response to the 'sis:student:messages' ENF message? Is it because messages show up in all those contexts? I don't remember the Class Enrollment card showing messages

Reply:
This is going to take over for CalGrant Acknowledgement expiry, which includes the MyAcademics and FinancialAid expiry portions.